scripts/asm-installer/install_asm

```diff
@@ -811,6 +844,10 @@ EOF
     fatal "Hub Workload Identity Pool is only supported for Mesh CA"
   fi
 
+  if [[ "${VM_EXPANSION}" -eq 1 && "${USE_HUB_WIP}" -eq 0 ]]; then
+    fatal "Hub Workload Identity Pool is required to add VM workloads. Run the script with the -o hub-meshca option."
+  fi
```

Comment:
Is there a reason we can't just automatically do this instead of making the user re-run the script?